### ISSUE TITLE
Crash when OCRing a long screenshot (User took a screenshot via Rocket)

### ISSUE THREAD

**Issue by cynthiatang** [mozilla-tw/ScreenshotGo#84]
Crash when OCRing [this screenshot ](https://drive.google.com/open?id=1b8yO4-BXPGqH9OnaMQ8iyVEg2VL1mtQx)

Video: https://drive.google.com/open?id=199Q7lwoiUTgH9LkjEgzzhcjQGgPi3yPJ

Reproduce rate: 10/10

Scryer 1.0(179).debug
Redmi 4X
Android 7.1.2

--------- beginning of crash
```
09-05 15:15:15.980 16718 17290 E AndroidRuntime: FATAL EXCEPTION: ForkJoinPool.commonPool-worker-5
09-05 15:15:15.980 16718 17290 E AndroidRuntime: Process: org.mozilla.scryer.debug, PID: 16718
09-05 15:15:15.980 16718 17290 E AndroidRuntime: java.lang.RuntimeException: Can't create handler inside thread that has not called Looper.prepare()
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at android.os.Handler.<init>(Handler.java:200)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at android.os.Handler.<init>(Handler.java:114)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at android.widget.Toast$TN$2.<init>(Toast.java:338)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at android.widget.Toast$TN.<init>(Toast.java:338)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at android.widget.Toast.<init>(Toast.java:103)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at android.widget.Toast.makeText(Toast.java:256)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at org.mozilla.scryer.detailpage.DetailPageActivity.processTextRecognitionResult(DetailPageActivity.kt:241)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at org.mozilla.scryer.detailpage.DetailPageActivity.access$processTextRecognitionResult(DetailPageActivity.kt:37)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at org.mozilla.scryer.detailpage.DetailPageActivity$startOcr$1.doResume(DetailPageActivity.kt:176)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at kotlin.coroutines.experimental.jvm.internal.CoroutineImpl.resume(CoroutineImpl.kt:42)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at kotlinx.coroutines.experimental.DispatchedTask$DefaultImpls.run(Dispatched.kt:150)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at kotlinx.coroutines.experimental.DispatchedContinuation.run(Dispatched.kt:14)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1383)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:256)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1123)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1961)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1909)
09-05 15:15:15.980 16718 17290 E AndroidRuntime: 	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:128)
09-05 15:15:15.993  2240  2270 I octvm_klo: klo lock
09-05 15:15:15.995  1513 10821 W ActivityManager:   Force finishing activity org.mozilla.scryer.debug/org.mozilla.scryer.detailpage.DetailPageActivity
09-05 15:15:15.997  2488  2635 I WtProcessController: Receive am_crash event for pid: 16718 pkg = org.mozilla.scryer.debug
09-05 15:15:16.001  1513 10821 D ActivityTrigger: ActivityTrigger activityPauseTrigger 
09-05 15:15:16.070  1513 17404 I Timeline: Timeline: App_transition_ready time:22717079
09-05 15:15:16.077  1513  1528 W BroadcastQueue: Skipping deliver [background] BroadcastRecord{5686aa9 u-1 android.net.conn.CONNECTIVITY_CHANGE} to ReceiverList{3145f73 16718 org.mozilla.scryer.debug/10129/u0 remote:b3adfe2}: process crashing
09-05 15:15:16.080  1513  1528 W BroadcastQueue: Skipping deliver [background] BroadcastRecord{d340865 u-1 android.net.conn.CONNECTIVITY_CHANGE} to ReceiverList{ccb7fcf 16718 org.mozilla.scryer.debug/10129/u0 remote:5ab182e}: process crashing
09-05 15:15:16.082 16718 16718 D HomeFragment: onStorageGranted
09-05 15:15:16.082 16718 16718 D HomeFragment: onOverlayGranted
09-05 15:15:16.082 16718 16718 D HomeFragment: onPermissionFlowFinish
09-05 15:15:16.092  1513  2593 I Timeline: Timeline: App_transition_ready time:22717102
09-05 15:15:16.159  2240  2270 I octvm_klo: get wanted event[mask:128, name:data_app_crash@1536153316159.txt] from the watchset
09-05 15:15:16.161  2488  2635 W KloServer: Aborted broadcast does not supprt for: org.mozilla.scryer.debug
09-05 15:15:16.161  2488  2635 I WtProcessController: MOVE TO FOREGROUND: org.mozilla.scryer.debug 10129
09-05 15:15:16.161  2488  2635 I WtProcessController: FOREGROUND INFO: name=org.mozilla.scryer.debug uid=10129 pid=16718 TaskId:834
09-05 15:15:16.166  2240  2270 I octvm_klo: start gathering logcat log...
09-05 15:15:16.167  2240  2270 I octvm_klo: gathering logcat log done
09-05 15:15:16.167  2240  2270 I octvm_klo: klo unlock
09-05 15:15:16.173  1513  1529 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@3534bf4
09-05 15:15:16.173  1513  1529 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@e53df1d
09-05 15:15:16.213  1735  1735 D PhoneStatusBar: disable: 0x00000200 -> 0x00000000 (diff: 0x00000200)
09-05 15:15:16.213  1735  1735 D PhoneStatusBar: disable: < expand icons alerts ticker system_info back home recent clock full hide keyguard none float >
09-05 15:15:16.236  1513  1529 I Timeline: Timeline: App_transition_ready time:22717246
09-05 15:15:16.238  1513  1529 I Timeline: Timeline: App_transition_ready time:22717248
09-05 15:15:16.240  1513 16314 I OpenGLRenderer: Initialized EGL, version 1.4
09-05 15:15:16.240  1513 16314 D OpenGLRenderer: Swap behavior 1
09-05 15:15:16.253  1513  1534 I Timeline: Timeline: App_transition_ready time:22717263
09-05 15:15:16.262  1735  1735 V PhoneStatusBarPolicy: updateManagedProfile mManagedProfileFocused: false mManagedProfileInQuietMode: false mKeyguardVisible: false mCurrentUserId:0 mCurrentProfileId:0 mSecondSpaceStatusIconVisible: true showIcon:false
09-05 15:15:16.280   695  1297 I miuiFirewall: miui firewall always allow android
09-05 15:15:16.288  1513 16314 D PerfShielderService: android|MiuiGlobalActions|234|22184537159869|207|5
09-05 15:15:16.290  2296  2307 D MiuiPerfServiceClient: interceptAndQueuing:1513|android|234|207|running|null
09-05 15:15:16.292  1735  1735 D PhoneStatusBar: disable: 0x00000000 -> 0x00000200 (diff: 0x00000200)
09-05 15:15:16.292  1735  1735 D PhoneStatusBar: disable: < expand icons alerts ticker system_info back home recent clock full hide keyguard none float >
09-05 15:15:16.292  1735  1735 D SimpleStatusBar: updateDarkMode() mDarkMode=true
09-05 15:15:16.292  1735  1735 D SimpleStatusBar: updateDarkMode() mIconsDarkArea=null
09-05 15:15:16.292   695  1297 I miuiFirewall: miui firewall always allow android
09-05 15:15:16.294  1735  1735 D SimpleStatusBar: updateBatteryChargingIcon() mQuickCharging=false; mDarkMode=true
09-05 15:15:16.337  1735  1735 D SimpleStatusBar: updateDarkMode() mDarkMode=false
09-05 15:15:16.337  1735  1735 D SimpleStatusBar: updateDarkMode() mIconsDarkArea=null
09-05 15:15:16.354  1735  1735 W ResourceType: No package identifier when getting value for resource number 0x00000000
09-05 15:15:16.355  1735  1735 E Icon    : Unable to load resource 0x00000000 from pkg=com.android.systemui
09-05 15:15:16.355  1735  1735 E Icon    : android.content.res.Resources$NotFoundException: Resource ID #0x0
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.content.res.ResourcesImpl.getValue(ResourcesImpl.java:201)
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.content.res.MiuiResourcesImpl.getValue(MiuiResourcesImpl.java:94)
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.content.res.Resources.getDrawable(Resources.java:788)
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.graphics.drawable.Icon.loadDrawableInner(Icon.java:316)
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.graphics.drawable.Icon.loadDrawable(Icon.java:272)
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.graphics.drawable.Icon.loadDrawableAsUser(Icon.java:380)
09-05 15:15:16.355  1735  1735 E Icon    : 	at com.android.systemui.statusbar.ExpandedIcon.getDrawable(ExpandedIcon.java:59)
09-05 15:15:16.355  1735  1735 E Icon    : 	at com.android.systemui.statusbar.StatusBarIconView.getIcon(StatusBarIconView.java:174)
09-05 15:15:16.355  1735  1735 E Icon    : 	at com.android.systemui.statusbar.StatusBarIconView.setIcon(StatusBarIconView.java:133)
09-05 15:15:16.355  1735  1735 E Icon    : 	at com.android.systemui.statusbar.StatusBarIconView.updateDarkMode(StatusBarIconView.java:266)
09-05 15:15:16.355  1735  1735 E Icon    : 	at com.android.systemui.statusbar.phone.SimpleStatusBar.updateDarkMode(SimpleStatusBar.java:291)
09-05 15:15:16.355  1735  1735 E Icon    : 	at com.android.systemui.statusbar.phone.PhoneStatusBar$19.run(PhoneStatusBar.java:3705)
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.os.Handler.handleCallback(Handler.java:754)
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.os.Handler.dispatchMessage(Handler.java:95)
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.os.Looper.loop(Looper.java:163)
09-05 15:15:16.355  1735  1735 E Icon    : 	at android.app.ActivityThread.main(ActivityThread.java:6205)
09-05 15:15:16.355  1735  1735 E Icon    : 	at java.lang.reflect.Method.invoke(Native Method)
09-05 15:15:16.355  1735  1735 E Icon    : 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
09-05 15:15:16.355  1735  1735 E Icon    : 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:794)
09-05 15:15:16.357  1735  1735 D SimpleStatusBar: updateBatteryChargingIcon() mQuickCharging=false; mDarkMode=false
09-05 15:15:16.522  1735  1735 D EventBus: [1735, u0] send(AppTransitionFinishedEvent)
09-05 15:15:16.522  1735  1735 D EventBus: [1735, u0]  -> ForcedResizableInfoActivityController [0xba317a2, P1] onBusEvent(AppTransitionFinishedEvent)
09-05 15:15:16.523  1735  1735 D EventBus: [1735, u0] onBusEvent(AppTransitionFinishedEvent) duration: 54 microseconds, avg: 908
09-05 15:15:16.570  1513  1534 I Timeline: Timeline: App_transition_stopped time:22717580
09-05 15:15:16.592  1513  1534 I Timeline: Timeline: Activity_windows_visible id: ActivityRecord{f257c8c u0 org.mozilla.scryer.debug/org.mozilla.scryer.MainActivity t834} time:22717602
09-05 15:15:16.595  1735  1735 D EventBus: [1735, u0] send(AppTransitionFinishedEvent)
09-05 15:15:16.595  1735  1735 D EventBus: [1735, u0]  -> ForcedResizableInfoActivityController [0xba317a2, P1] onBusEvent(AppTransitionFinishedEvent)
09-05 15:15:16.596  1735  1735 D EventBus: [1735, u0] onBusEvent(AppTransitionFinishedEvent) duration: 65 microseconds, avg: 905
09-05 15:15:16.626   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:16.628   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:16.628   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:16.628   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:16.628   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
09-05 15:15:16.628   672   764 D ThermalEngine: sensor_wait: case_therm Wait start. 1000ms
09-05 15:15:16.687  1513 17378 W ErrorReport: Fail to sendHttpRequest
09-05 15:15:16.687  1513 17378 W ErrorReport: java.lang.IllegalArgumentException: HTTP entity may not be null
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at org.apache.http.util.EntityUtils.toString(EntityUtils.java:115)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at org.apache.http.util.EntityUtils.toString(EntityUtils.java:151)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at miui.util.ErrorReport.tQ(SourceFile:396)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at miui.util.ErrorReport.sendReportRequest(SourceFile:353)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at miui.util.ErrorReport$1.wy(SourceFile:369)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at miui.util.ErrorReport$1.doInBackground(SourceFile:368)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at android.os.AsyncTask$2.call(AsyncTask.java:305)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:243)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
09-05 15:15:16.687  1513 17378 W ErrorReport: 	at java.lang.Thread.run(Thread.java:760)
09-05 15:15:16.689  1513 17378 W ContextImpl: Calling a method in the system process without a qualified user: android.app.ContextImpl.bindService:1458 android.content.ContextWrapper.bindService:646 miui.os.DropBoxManager.yG:361 miui.os.DropBoxManager.yF:350 miui.os.DropBoxManager.addText:314 
09-05 15:15:16.956  1765  1765 D wpa_supplicant: wlan0: Control interface command 'SIGNAL_POLL'
09-05 15:15:16.960  1765  1765 D wpa_supplicant: CTRL-DEBUG: global_ctrl_sock-sendto: sock=8 sndbuf=212992 outq=0 send_len=48
09-05 15:15:17.425   802   802 I MSM-irqbalance: Decided to move IRQ3 from CPU2 [P:0] to CPU4 [P:1] (banned)
09-05 15:15:17.426   802   802 I MSM-irqbalance: Decided to move IRQ5 from CPU2 [P:0] to CPU6 [P:1] (banned)
09-05 15:15:17.426   802   802 I MSM-irqbalance: Decided to move IRQ3 from CPU0 [P:0] to CPU5 [P:1] (banned)
09-05 15:15:17.427   802   802 I MSM-irqbalance: Decided to move IRQ5 from CPU0 [P:0] to CPU7 [P:1] (banned)
09-05 15:15:17.628   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:17.630   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:17.630   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:17.630   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:17.630   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
09-05 15:15:17.630   672   764 D ThermalEngine: sensor_wait: case_therm Wait start. 1000ms
09-05 15:15:18.631   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:18.632   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:18.632   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:18.632   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:18.632   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
09-05 15:15:18.632   672   764 D ThermalEngine: sensor_wait: case_therm Wait start. 1000ms
09-05 15:15:19.633   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:19.634   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:19.634   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:19.634   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:19.634   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
09-05 15:15:19.634   672   764 D ThermalEngine: sensor_wait: case_therm Wait start. 1000ms
09-05 15:15:19.966  1765  1765 D wpa_supplicant: wlan0: Control interface command 'SIGNAL_POLL'
09-05 15:15:19.986  1765  1765 D wpa_supplicant: CTRL-DEBUG: global_ctrl_sock-sendto: sock=8 sndbuf=212992 outq=0 send_len=48
09-05 15:15:20.635   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:20.636   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:20.636   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:20.636   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:20.636   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
09-05 15:15:20.636   672   764 D ThermalEngine: sensor_wait: case_therm Wait start. 1000ms
09-05 15:15:20.929 25893 25945 D ClClient: Not sending keepalive.  Current connection state=STOPPED
09-05 15:15:21.637   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:21.638   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:21.639   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:21.639   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:21.639   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
09-05 15:15:21.639   672   764 D ThermalEngine: sensor_wait: case_therm Wait start. 1000ms
09-05 15:15:22.426   802   802 I MSM-irqbalance: Decided to move IRQ5 from CPU0 [P:0] to CPU4 [P:1] (banned)
09-05 15:15:22.426   802   802 I MSM-irqbalance: Decided to move IRQ3 from CPU0 [P:0] to CPU6 [P:1] (banned)
09-05 15:15:22.639   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:22.641   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:22.641   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:22.641   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:22.641   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
09-05 15:15:22.641   672   764 D ThermalEngine: sensor_wait: case_therm Wait start. 1000ms
09-05 15:15:22.991  1765  1765 D wpa_supplicant: wlan0: Control interface command 'SIGNAL_POLL'
09-05 15:15:23.013  1765  1765 D wpa_supplicant: CTRL-DEBUG: global_ctrl_sock-sendto: sock=8 sndbuf=212992 outq=0 send_len=48
09-05 15:15:23.642   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:23.643   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:23.643   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:23.643   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:23.643   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
09-05 15:15:23.643   672   764 D ThermalEngine: sensor_wait: case_therm Wait start. 1000ms
09-05 15:15:24.644   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:24.645   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:24.645   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:24.645   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:24.645   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
09-05 15:15:24.645   672   764 D ThermalEngine: sensor_wait: case_therm Wait start. 1000ms
09-05 15:15:25.646   672   764 D ThermalEngine: sensor_wait: case_therm Wait done.
09-05 15:15:25.647   672   764 D ThermalEngine: sensor_monitor: case_therm Reading 34000 .
09-05 15:15:25.647   672   764 I ThermalEngine: Sensor:case_therm:34000 mC
09-05 15:15:25.648   672   764 D ThermalEngine: update_active_thresh: case_therm Active(1), Hi(1) 42000, Lo(0) -2147483648, Interval(1) 1000
09-05 15:15:25.648   672   764 D ThermalEngine: sensor_monitor: case_therm Sensor wait.
```

**Comment by weslyhuang** [comment]
crash so S1, and maybe P1 as well? Or we need some restricting about OCR-ing a long screenshot?

**Comment by weslyhuang** [comment]
this new crash is fixed by #442
